package/src/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts

```diff
@@ -21,26 +21,33 @@ export const getChannelPreviewDisplayName = <
   channelName?: string;
   currentUserId?: string;
   members?: Channel<StreamChatGenerics>['state']['members'];
-}) => {
+}): string => {
```

Comment:
Let's increase the overall readability of this func please!

Reply:
I think with the current refactor the function is a bit more  clearer

Reply:
@madsroskar is this suffecient?

Reply:
I'd drop the inline comments, as they don't add any value as they are at the moment IMHO. It's better though, still have a couple of nits I'll add in comments. I can clearer read what's going on in the function now at least